website/integrations/services/proxmox-ve/index.md

```diff
@@ -24,36 +24,63 @@ The following placeholders will be used:
 -   `proxmox.company` is the FQDN of the Proxmox VE server.
 -   `authentik.company` is the FQDN of the authentik install.
 
-### Step 1
+## authentik setup
 
-Under _Providers_, create an OAuth2/OpenID provider with these settings:
+1. **Navigate to the Admin interface, then click Providers to create an OAuth2/OpenID provider with these settings:**
 
--   Name: proxmox
--   Redirect URI: `https://proxmox.company:8006` (Note the absence of the trailing slash, and the inclusion of the webinterface port)
--   Signing Key: Select any available key
+    - **Name:** proxmox
+    - **Redirect URI:** `https://proxmox.company:8006` (No trailing slash, include the web interface port)
+    - **Signing Key:** Select any available key
 
-### Step 2
+2. **Create an application using this provider**
+    - Under **Applications** > **Applications** in the Admin interface, create a new application and configure it to use the provider created in the previous step.
+    - Optionally, apply access restrictions to the application.
+    - Set the **Launch URL** to `https://proxmox.company:8006`.
 
-Create an application which uses this provider. Optionally apply access restrictions to the application.
+## Proxmox VE setup (web interface)
```

Comment:
```suggestion
## Proxmox VE (web interface) configuration
```

Reply:
Is "(web interface)" describing the "Proxmox VE"? Hopefully I didn't mess things up by reordering...?